terraform/src/export.tf

```diff
@@ -0,0 +1,107 @@
+resource "aws_lambda_function" "exportPassLambda" {
+  function_name = "exportPass"
+
+  filename         = "artifacts/exportPass.zip"
+  source_code_hash = filebase64sha256("artifacts/exportPass.zip")
+
+  handler = "lambda/exportPass/index.handler"
+  runtime = "nodejs14.x"
+  timeout = 30
+
+  environment {
+    variables = {
+      TABLE_NAME                   = data.aws_ssm_parameter.db_name.value,
+      JWT_SECRET                   = local.jwtSecret.jwtSecret,
+      PUBLIC_FRONTEND              = data.aws_ssm_parameter.public_url.value,
+      GC_NOTIFY_API_PATH           = data.aws_ssm_parameter.gc_notify_api_path.value,
+      GC_NOTIFY_API_KEY            = data.aws_ssm_parameter.gc_notify_api_key.value,
+      GC_NOTIFY_CANCEL_TEMPLATE_ID = data.aws_ssm_parameter.gc_notify_cancel_template_id.value,
+      PASS_CANCELLATION_ROUTE      = data.aws_ssm_parameter.pass_cancellation_route.value,
+      S3_BUCKET_DATA               = data.aws_ssm_parameter.s3_bucket_data.value,
+    }
+  }
+
+  role = aws_iam_role.exportRole.arn
+}
+
+resource "aws_api_gateway_resource" "exportPassResource" {
+  rest_api_id = aws_api_gateway_rest_api.apiLambda.id
+  parent_id   = aws_api_gateway_rest_api.apiLambda.root_resource_id
+  path_part   = "export-pass"
+}
+
+// Defines the HTTP GET /pass API
+resource "aws_api_gateway_method" "exportPassMethod" {
+  rest_api_id   = aws_api_gateway_rest_api.apiLambda.id
+  resource_id   = aws_api_gateway_resource.exportPassResource.id
+  http_method   = "GET"
+  authorization = "NONE"
+}
+
+// Integrates the APIG to Lambda via POST method
+resource "aws_api_gateway_integration" "exportPassIntegration" {
+  rest_api_id = aws_api_gateway_rest_api.apiLambda.id
+  resource_id = aws_api_gateway_resource.exportPassResource.id
+  http_method = aws_api_gateway_method.exportPassMethod.http_method
+
+  integration_http_method = "POST"
+  type                    = "AWS_PROXY"
+  uri                     = aws_lambda_function.exportPassLambda.invoke_arn
+}
+
+resource "aws_lambda_permission" "exportPassPermission" {
+  statement_id  = "AllowParksDayUsePassAPIInvoke"
+  action        = "lambda:InvokeFunction"
+  function_name = aws_lambda_function.exportPassLambda.function_name
+  principal     = "apigateway.amazonaws.com"
+  source_arn    = "${aws_api_gateway_rest_api.apiLambda.execution_arn}/*/GET/pass"
```

Comment:
This should be `${aws_api_gateway_rest_api.apiLambda.execution_arn}/*/GET/export-pass`

Reply:
Fixed in 160b525